client/src/components/commonComponents/Input/index.css

```diff
@@ -0,0 +1,12 @@
+input {
```

Comment:
don't use the tag name in the CSS

Reply:
I removed the names of the tags and put classes instead